src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h

```diff
@@ -149,8 +149,6 @@ extern "C" {
 #define MKV_MULAW_CONTENT_TYPE            ((PCHAR) "audio/mulaw")
 #define MKV_AVC_CONTENT_TYPE              ((PCHAR) "video/avc")
 #define MKV_HEVC_CONTENT_TYPE             ((PCHAR) "video/hevc")
-#define MKV_H264_AAC_MULTI_CONTENT_TYPE   ((PCHAR) "video/h264,audio/aac")

```

Comment:
I am assuming we will be doing the append operation within the SDK for this, correct?

Reply:
You will need to change this because it's used here:

https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/master/src/mkvgen/tst/MkvgenApiFunctionalityTest.cpp#L920

You'll have to pass in the right thing there.

Reply:
Perhaps we can keep this for backwards comp since this is in our public include.  But in our new APIs we can always construct the content type string by joining the video and audio types

Reply:
In that case you don't necessarily need to change the test however we should probably have tests at some point with supported combinations of video and audio content types.